pkg/kubelet/kubelet.go

```diff
@@ -1256,6 +1257,9 @@ func (kl *Kubelet) setupDataDirs() error {
 	if err := os.MkdirAll(kl.getPluginsDir(), 0750); err != nil {
 		return fmt.Errorf("error creating plugins directory: %v", err)
 	}
+	if err := os.MkdirAll(kl.getPodResourcesDir(), 0750); err != nil {
```

Comment:
This should probably part of an `Init` function in the server rather than being at the top level

Reply:
I think I prefer keeping it with the creation of all of the other directories the kubelet makes.